lib/ballot_test.go

```diff
@@ -0,0 +1,96 @@
+package sebak
+
+import (
+	"testing"
+	"time"
+
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/stellar/go/keypair"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/node"
+)
+
+func TestBallotBadConfirmedTime(t *testing.T) {
+	kp, _ := keypair.Random()
+	endpoint, _ := sebakcommon.NewEndpointFromString("https://localhost:1000")
+	node, _ := sebaknode.NewLocalNode(kp, endpoint, "")
+
+	round := Round{Number: 0, BlockHeight: 0, BlockHash: "", TotalTxs: 0}
+
+	{
+		ballot := NewBallot(node, round, []string{})
+		ballot.Sign(kp, networkID)
+
+		if err := ballot.IsWellFormed(networkID); err != nil {
+			t.Error(err)
+			return
+		}
+	}
+
+	{ // bad `Ballot.B.Confirmed` time; too ahead
```

Comment:
I think those should be their own test instead of anon blocks

Reply:
@Geod24 Separating different test also looks good, but the cases is limited and entire block is also short, this code can be acceptable, I think.